### PR TITLE
Adding EnTK to conda

### PR DESCRIPTION
**Pull request by vivek-bala**
A conda recipe for EnTK.

### CODE REVIEW

**Comment by vivek-bala** [comment]
@iparask @andre-merzky will you be able to act as maintainers for this package?

**Comment by iparask** [comment]
Yes.

**Comment by andre-merzky** [comment]
yes.

File: recipes/radical.entk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed  .
```
Comment (marcelotrevisani) [inline recipes/radical.entk/meta.yaml]: Could you please change that to

```
  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

Comment (vivek-bala) [reply recipes/radical.entk/meta.yaml]: Done

**Comment by marcelotrevisani** [comment]
LGTM!

Waiting the CI to finish!

Thanks to put this in ``conda-forge``!  :)

**Comment by iparask** [comment]
Hello @marcelotrevisani, there a feedstock repo under conda-forge. I think it might need some time before it gets populated. That was the case with the two other packages we have `radical.pilot` and `saga-python`. I suggest that we wait before creating a new pull request.

**Comment by marcelotrevisani** [comment]
I reverted that, so it will not be created

Just create another PR and I will personally take a look, maybe it was just an error in the CI, but just to be sure that everything is ok. Could you please create another PR?

**Comment by iparask** [comment]
I am asking because it does exist: https://github.com/conda-forge/radical.entk-feedstock and I am not sure if it is okay.

Either way we will create a new one! Thank you!

**Comment by jakirkham** [comment]
There is an ongoing issue with conversion. ( https://github.com/conda-forge/staged-recipes/issues/6659 )

**Comment by iparask** [comment]
@jakirkham was #6659 resolved? I still see a `radical.entk-feedstock` repo which is empty. In addition, we opened another PR (#6654) based on @marcelotrevisani instructions. In that PR the linter objects since the feedstock repo exists.

Thank you!

**Pull request merged**